src/resources/fundingLogos/credit.jsx

```diff
@@ -42,7 +42,11 @@ export function creditLogo({ logoColor, locale } : { logoColor : $Values<typeof
             </svg>
         );
     default:
-        return (
+        return isFundingThrottleEnabled ? (
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 53 11">
```

Comment:
Can you please specify width and height? We've seen SVGs act funky on older browsers without this.

Reply:
Done, though the width is 964px and the height is 200px.  Just the FYI.

Reply:
OK. I guess the main thing is, if you can just make sure it renders without any problems in IE11 we should be ok.

Reply:
Tested rendering the SVG on IE11 with no issues.